packages/graph/hash_graph/lib/graph/src/types/schema/data_type.rs

```diff
@@ -26,6 +30,19 @@ impl DataTypeReference {
     }
 }
 
+impl ValidateUri for DataTypeReference {
+    fn validate_uri(&self, base_uri: &BaseUri) -> Result<(), ValidationError> {
+        ensure!(
+            base_uri == self.reference.base_uri(),
+            ValidationError::BaseUriMismatch {
+                base_uri: base_uri.clone(),
+                versioned_uri: self.reference.clone()
```

Comment:
Didn't we change this to self.uri for PropertyTypeReference? I thought we had done for DataTypeReference too but I must have missed it. Can we drive-by?

Reply:
I now see that we actually don't seem to have it for `PropertyTypeReference` _or_ `DataTypeReference. I think you force pushed and removed the commits from #790 🙃

Reply:
The commit links are broken in your previous messages:
https://github.com/hashintel/hash/pull/790#discussion_r919983087

Reply:
For clarification, #790 only renamed the methods, not the functions (which were on purpose). Off GH we discussed this and decided to also rename the fields but leaving `DataType::id`, `PropertyType::id`, and `EntityType::id`.